orientdb/src/main/java/com/yahoo/ycsb/db/OrientDBClient.java

```diff
@@ -134,13 +147,22 @@ public void init() throws DBException {
       }
     }
 
-    System.err.println("OrientDB connection created with " + url);
+    log.info("OrientDB connection created with " + url);
     dictionary = db.getMetadata().getIndexManager().getDictionary();
     if (!db.getMetadata().getSchema().existsClass(CLASS)) {
       db.getMetadata().getSchema().createClass(CLASS);
     }
 
-    db.declareIntent(new OIntentMassiveInsert());
+    if (intent.equals(ORIENTDB_MASSIVEINSERT)) {
```

Comment:
No, I don't think they should be set automatically, that is actually why I moved this to a user property. It is an optimization, not a necessary action. Prior to this change MassiveInsert was being set no matter what which didn't much make since for workloads that weren't inserting.